homeassistant/components/toon/const.py

```diff
@@ -112,6 +113,15 @@
 }
 
 SENSOR_ENTITIES = {
+    "current_display_temperature": {
+        ATTR_NAME: "Temperature",
+        ATTR_SECTION: "thermostat",
+        ATTR_MEASUREMENT: "current_display_temperature",
+        ATTR_UNIT_OF_MEASUREMENT: TEMP_CELSIUS,
+        ATTR_DEVICE_CLASS: DEVICE_CLASS_TEMPERATURE,
+        ATTR_ICON: None,
+        ATTR_DEFAULT_ENABLED: True,
```

Comment:
Let's not enable it by default, since it's a duplicate ?

Reply:
I also wonder if we should just create a generic "sensorify" integration that can extract relevant sensors from climate, weather etc integrations.

Reply:
Disabled it by default now.

About your sensorify comment, I think that is to complicated in thinking. If we go such a route, I would turn it around.

A climate, weather component is built upon existing sensor entities, because that is what they are. They combine multiple data into a single entity.